domains/seldomgreen.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "seldomgreeen",
+           "email": "72107155+seldomgreeen@users.noreply.github.com",
```

Comment:
Invalid email, we need a email that we are able to contact you, not a no reply email.